Source/WebKit/Shared/WebCoreArgumentCoders.cpp

```diff
@@ -650,6 +651,39 @@ bool ArgumentCoder<ResourceError>::decode(Decoder& decoder, ResourceError& resou
     return true;
 }
 
+void ArgumentCoder<SecurityOriginData>::encode(Encoder& encoder, const SecurityOriginData& data)
```

Comment:
Ideally, these would be data members on SecurityOriginData so that we wouldn't have to include the large WebCoreArgumentCoders.h in some places. You can use ObjectIdentifier.h as an example.